client/src/components/AppointmentCreateForm.js

```diff
@@ -133,7 +126,7 @@ const AppointmentCreateForm: () => React$Node = () => {
       </div>
       </form>   
       {mutationLoading && <p>Loading...</p>}
-      {mutationError && <p>Error:( Please try again</p>}
+      {mutationError && <p>Error:( We can not accept this appointment.</p>}
```

Comment:
This sounds like this appointment is invalid. For each error, it's either the user's fault or the server's fault; your message make it sound like it's the user's fault.

Reply:
I just changed it to "Error:( Sorry, something wrong happened." SInce for now we all do not know what reasons would cause the error to trigger. Does this new warning sentence make a sense to you?

Reply:
Yes that works.